block/config.ml

```diff
@@ -2,8 +2,10 @@ open Mirage
 
 let () =
   let main = foreign "Unikernel.Block_test" (console @-> block @-> job) in
-  let img =
-    match get_mode () with
+  let img = match get_mode () with
     | `Xen -> block_of_file "xvda1"
-    | `Unix -> block_of_file "disk.img" in
-  register "block_test" [main $ default_console $ img]
+    | `Unix | `MacOSX -> block_of_file "disk.img"
+  in
+  register "block_test" [
+    main $ default_console $ img
+  ]
```

Comment:
In half the examples, it's 
```ocaml
register "bla" [
  thing
]
``` 

In the other half it's 
```ocaml
register "bla" [thing]
```

Decide yourself! :p